fastapi_pagination/default.py

```diff
@@ -59,13 +59,15 @@ def create(
         if not isinstance(params, Params):
             raise ValueError("Page should be used with Params")
 
-        pages = ceil(total / params.size) if total is not None else None
+        size = params.size if params.size is not None else total
+        page = params.page if params.page is not None else 1
```

Comment:
@uriyyo 
Isn't `page = params.page or 1` better in this case ? As page 0 should not be accepted ?
Same for size ?

Reply:
@s-rigaud Thanks for your suggestion.

We can have custom params that can use zero-based pagination. `page = params.page or 1` can produce bugs, cause we never now how end-user customization will look like.

Reply:
Oh ok I understand, thanks for the response 👍